curl_cffi/requests/headers.py

```diff
@@ -11,8 +11,6 @@
     typing.Mapping[bytes, bytes],
     typing.Sequence[typing.Tuple[str, str]],
     typing.Sequence[typing.Tuple[bytes, bytes]],
-    typing.Sequence[str],
```

Comment:
`list[str|bytes]` is actually used in the `__init__` function. See line 90

Reply:
Done